src/nupic/math/Array2D.hpp

```diff
@@ -36,6 +36,8 @@
 // try to separate algorithms and write them for all types of matrices??
 // optimize kernels depending on cache sizes?
 
+// Note: cannot be made a nupic::Serializable class because it is not in nupic namespace.
```

Comment:
can you add it to ns nupic:: ?

Reply:
Well...perhaps but lets make that a separate PR.  It could get messy.